### PR TITLE
Wishlist

### PR DESCRIPTION
**Pull request by piyushsh25**
https://deploy-preview-5--zevnon.netlify.app/docs/pages/wishlist.html

### CODE REVIEW

File: docs/css/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+.price-summary .h5 {
+    color: var(--color-success);
+}
+
+.button.outline {
+    margin: 0;
+    padding: 0;
+}
+
+.h4 {
```
Comment (Shivanshsharma01) [inline docs/css/wishlist.css:75]: It's good to delete commented code

File: docs/css/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+}
+
+.product-content {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.h4 {
+    margin-left: var(--size10);
+    color: rgb(4, 55, 122);
```
Comment (Shivanshsharma01) [inline docs/css/wishlist.css:30]: can use variable here

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean and well structured. Good to merge. Can implement suggested changes

**Review by RakshithBhat03 — APPROVED** [review]
The code looks good. Please implement the changes suggested by the other reviewer.

**Comment by piyushsh25** [comment]
Worked on the suggestions. Merging now.

**Pull request merged**